Apps/W1/Shopify/app/src/Metafields/Pages/ShpfyMetafields.Page.al

```diff
@@ -91,9 +91,18 @@ page 30163 "Shpfy Metafields"
     internal procedure RunForResource(ParentTableId: Integer; OwnerId: BigInteger; ShopCode: Code[20])
     var
         Metafield: Record "Shpfy Metafield";
+        OwnerType: Enum "Shpfy Metafield Owner Type";
     begin
         Shop.Get(ShopCode);
-        IsPageEditable := (Shop."Sync Item" = Shop."Sync Item"::"To Shopify") and (Shop."Can Update Shopify Products");
+
+        case Metafield.GetOwnerType(ParentTableId) of
```

Comment:
Maybe this should be an additional procedure on the interface?
Since this would likely have to be extended again when the next owner type woudl be added...

Reply:
added procedure to interface